core/src/test/scala/org/apache/spark/scheduler/CoarseGrainedSchedulerBackendSuite.scala

```diff
@@ -129,4 +181,34 @@ class CoarseGrainedSchedulerBackendSuite extends SparkFunSuite with LocalSparkCo
       { return }
     )
   }
+
+  private def getCustomExecutorLogUrl(includeFileName: Boolean): String = {
+    val baseUrl = "http://newhost:9999/logs/clusters/{{CLUSTER_ID}}/users/{{USER}}/containers/" +
+      "{{CONTAINER_ID}}"
+    if (includeFileName) baseUrl + "/{{FILE_NAME}}" else baseUrl
+  }
+
+  private def getTestExecutorLogUrls: Map[String, String] = Map(
+    "stdout" -> "http://oldhost:8888/logs/dummy/stdout",
+    "stderr" -> "http://oldhost:8888/logs/dummy/stderr")
+
+  private def getTestExecutorAttributes: Map[String, String] = Map(
+    "CLUSTER_ID" -> "cl1",
+    "USER" -> "dummy",
+    "CONTAINER_ID" -> "container1",
+    "LOG_FILES" -> "stdout,stderr"
+  )
+
+  private def getExpectedCustomExecutorLogUrl(
```

Comment:
Only called from one place, could be inlined.

Reply:
Ping? It's also always called with a file name set. Inlining would make this much smaller.

Reply:
Just missed it. Fixed.